package com.sesung.exception;

public class ExceptionTest2 {

	public static void main(String[] args){
		Ex1 ex = new Ex1();
		try{
			ex.test();
		}catch (Exception e) {
			
		}

	}
}
